src/layouts/docs/utils/adjust-link-urls-plugin.ts

```diff
@@ -0,0 +1,14 @@
+import { visit } from 'unist-util-visit'
+import type { Root } from 'mdast'
+
+export default function adjustLinkUrlPlugin({
```

Comment:
This plugin can be used to adjust the `url` values of links in markdown content. Currently only used for Vault's `/api` link issue.

Reply:
nit: this might be easier to discover if it were closer to the `src/` root...maybe `src/remark-plugins` or `src/lib/remark-plugins`?